support-config/src/main/resources/reference.conf

```diff
@@ -5,3 +5,7 @@ include classpath("touchpoint.PROD.conf")
 
 get-address-io-api.url = "https://api.getAddress.io/v2/uk/"
 get-address-io-api.key = "hyj46rv0Uk2D1BtcW5M8jQ17434" //Test key, overridden in PROD
+AVCalculator {
```

Comment:
in case anyone else wonders, this is expected by a dependency - https://github.com/guardian/acquisitions-value-calculator-client